### PR TITLE
MISUV-3863 Show cutOver credit status

### PR DESCRIPTION
**Pull request by andriikondiuk**

### CODE REVIEW

File: app/models/financialDetails/DocumentDetail.scala [hunk 1]
```diff
@@ -96,7 +96,7 @@ case class DocumentDetail(taxYear: String,
     else isPaid
   }
 
-  val isPartPaid: Boolean = outstandingAmount.getOrElse[BigDecimal](0) != originalAmount.getOrElse[BigDecimal](0)
+  val isPartPaid: Boolean = outstandingAmount.getOrElse[BigDecimal](0).abs != originalAmount.getOrElse[BigDecimal](0).abs
```
Comment (avelx) [inline app/models/financialDetails/DocumentDetail.scala:99]: what would happen if one value negative and one positive?
I would rather add unit test to see various scenarios

**Review by avelx — CHANGES_REQUESTED** [review]

File: app/models/financialDetails/DocumentDetail.scala (hunk 1 above)
Comment (andriikondiuk) [reply app/models/financialDetails/DocumentDetail.scala:99]: Agree it's a valid question. thanks

Comment (andriikondiuk) [reply app/models/financialDetails/DocumentDetail.scala]: I've removed the .abs and added a unit tests, thanks

File: test/testConstants/FinancialDetailsTestConstants.scala [hunk 2]
```diff
@@ -938,15 +938,15 @@ object FinancialDetailsTestConstants {
       outstandingAmount = Some(BigDecimal(-500.00)),
       paymentLotItem = None,
       paymentLot = None,
-      originalAmount = Some(BigDecimal(800.00)),
+      originalAmount = Some(BigDecimal(-800.00)),
```
Comment (avelx) [inline test/testConstants/FinancialDetailsTestConstants.scala:941]: changes in this file seems to be not relevant ( if you rollback them back test would not fail)

**Review by avelx — APPROVED** [review]
looks good

**Pull request merged**